templates/service-template-rosa.yml

```diff
@@ -0,0 +1,362 @@
+  value: "1Gi"
+
+- name: CPU_REQUEST
+  description: CPU request for the API pods.
+  value: "200m"
+
+- name: CPU_LIMIT
+  description: CPU limit for the API pods.
+  value: "1"
+
+- name: SERVER_REPLICAS
+  description: Number of replicas of the service to run.
+  value: "3"
+
+- name: ENABLE_JWT
+  displayName: Enable JWT
+  description: Enable JWT authentication validation
+  value: "false"
+
+- name: ENABLE_HTTPS
+  displayName: Enable HTTPS
+  description: Enable HTTPS rather than HTTP
+  value: "false"
+
+- name: HTTP_SERVER_BINDPORT
+  displayName: HTTP Server Bindport
+  description: HTTP server bind port
+  value: "8000"
+
+- name: ENABLE_GRPC_SERVER
+  displayName: Enable gRPC Server
+  description: Enable gRPC server
+  value: "true"
+
+- name: GRPC_SERVER_BINDPORT
+  displayName: gRPC Server Bindport
+  description: gRPC server bind port
+  value: "8090"
+
+- name: METRICS_SERVER_BINDPORT
+  displayName: Metrics Server Bindport
+  description: Metrics server bind port
+  value: "8080"
+
+- name: HEALTH_CHECK_SERVER_BINDPORT
+  displayName: Health check Server Bindport
+  description: Health check server bind port
+  value: "8083"
+
+- name: HTTP_SERVER_HOSTNAME
+  displayName: HTTP Server Hostname
+  description: Server's public hostname
+  value: ""
+
+- name: ENABLE_AUTHZ
+  displayName: Enable Authz
+  description: Enable Authorization on endpoints, should only be disabled for debug
+  value: "true"
+
+- name: DB_MAX_OPEN_CONNS
+  displayName: Maximum Open Database Connections
+  description: Maximum number of open database connections per pod
+  value: "50"
+
+- name: DB_SSLMODE
+  displayName: DB SSLmode
+  description: Database ssl mode (disable | require | verify-ca | verify-full)
+  value: "verify-full"
+
+- name: ENABLE_DB_DEBUG
+  displayName: Enable DB Debug
+  description: framework's debug mode
+  value: "false"
+
+- name: ENABLE_METRICS_HTTPS
+  displayName: Enable Metrics HTTPS
+  description: Enable HTTPS for metrics server
+  value: "false"
+
+- name: ENABLE_OCM_MOCK
+  displayName: Enable OCM Mock
+  description: Enable mock uhc client
+  value: "true"
+
+- name: HTTP_READ_TIMEOUT
+  displayName: HTTP Read Timeout
+  description: HTTP server read timeout
+  value: 5s
+
+- name: HTTP_WRITE_TIMEOUT
+  displayName: HTTP Write Timeout
+  description: HTTP server write timeout
+  value: 30s
+
+- name: LABEL_METRICS_INCLUSION_DURATION
+  displayName: Label metrics inclusion duration
+  description: A cluster's last telemetry date needs be within in this duration in order to have labels collected
+  value: "168h"
+
+- name: MAESTRO_SERVER_EXPOSURE
+  displayName: The type of service to be used to expose the Maestro Server
+  value: "ClusterIP"
+
+objects:
+  - kind: Deployment
+    apiVersion: apps/v1
+    metadata:
+      name: maestro
+      labels:
+        app: maestro
+    spec:
+      selector:
+        matchLabels:
+          app: maestro
+      replicas: ${{SERVER_REPLICAS}}
```

Comment:
no error for 2 braces?

Reply:
no errors, it works, and I will use 1 brace to keep same code style with others